data/categories.txt

```diff
@@ -6093,24 +6093,133 @@ sw:Bwawa la kuogelea
 fa:استخرشنا
 mr:जलतरण तलाव
 
-sport-equestrian
-en:Equestrian Sports
-ru:Конный спорт
-bg:Коне|спорт|езда
-nl:Paardensport|ruitersport
-ko:승마|마술 스포츠
-pl:Jazda konna
-fr:Sport équestre
-hu:Lovassportok
-de:Pferdesport
-it:Equitazione
-es:Deportes ecuestres
-eu:Zaldi-kirolak
-pt:Desportos equestres
-pt-BR:Esportes equestres
-tr:Binicilik
-el:Ιππασία
-mr:घोडेस्वार खेळ
+sport-american_football
+en:American Football
+ar:كرة القدم الأمريكية
+cs:americký fotbal
```

Comment:
The case should the same as for other translations.

Reply:
Case doesn't really matter for `categories.txt`.
Hopefully one day we have an autoformatter for it!